auth-web/src/views/auth/AccountSettings.vue

```diff
@@ -280,8 +280,9 @@ export default class AccountSettings extends Mixins(AccountMixin) {
   private readonly permissions!: string[]
 
   private handleBackButton (): void {
-    const backTo = this.isStaff ? Pages.STAFF_DASHBOARD : `/account/${this.orgId}/business`
-    this.$router.push(backTo)
+    this.isStaff
+      ? this.$router.push(Pages.STAFF_DASHBOARD)
+      : window.location.replace(ConfigHelper.getBcrosDashboardURL())
```

Comment:
Hmm, wait. I think staff cannot access the BCROS Dashboard.

Users that have a "staff" role, when they login from BCROS, are redirected to the Business Registry staff dashboard.

(There is another type of "staff" but they don't have staff role and you don't have to worry about them in Business Registries code.)

Reply:
Staff will route to the Staff dashboard (local Dashboard)
Otherwise it will goto the Uber Dashboard.

Reply:
Oops I saw that backwards. You are correct. Closing this conversation.